src/MessageReporterAwareTrait.php

```diff
@@ -16,9 +16,10 @@ trait MessageReporterAwareTrait {
 	protected $messageReporter;
 
 	/**
+	 * @param MessageReporter $messageReporter
+	 *
 	 * @since 1.3
 	 *
-	 * @param MessageReporter $messageReporter
 	 */
```

Comment:
Not sure why, but `since` comes always before `param`/`return`/`throws` so some linting rules seems off here.

Reply:
The CS does not care. For some reason my IDE did that reorder. Fixed now.

Reply:
I know you are not a big fan of these comments but I find them helpful when something was introduced without having to consult `git blame` etc.

Reply:
Pff it also changed in other files. I'd rather not care about that as hopefully soon this can use PHP 7 and those type hints in the docs can be ditched

Reply:
Fun fact: it was me that introduced those @since comments into MediaWiki world. In the last few years I am no longer adding them within classes with some rare exceptions. The tags on the class level suffice to not need to do git blame.